container/root/run.d/99-phpfpm.sh

```diff
@@ -17,6 +17,9 @@ sed -i "s/pm.max_spare_servers = [0-9]\+/pm.max_spare_servers = 128/" $PHPFPM_CO
 
 sed -i "s/;pm.max_requests = [0-9]\+/pm.max_requests = 1024/" $PHPFPM_CONF
 
+# Ensure environment variables aren't cleaned, will make it into FPM  workers
+sed -i "s/;clear_env/clear_env/" $PHPFPM_CONF
```

Comment:
just double-checking, will this match more than one occurrence of this pattern? do we need a `/g` to make sure it matches more?